tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
+    # shellcheck disable=SC2046
+    export $(grep -v '^#' .env | xargs -d '\n')
+fi
+
+MAKE='make --debug=v --output-sync'
+$MAKE IMAGES="ceph" build
+build_Image=$(docker images | grep build | awk '{print $1; exit}')
+git_hash=$(git rev-parse --short "${GITHUB_SHA}")
+tag_Image=quay.io/paarora/rook-ceph-operator:v${BRANCH_NAME}$git_hash
```

Comment:
Is this the tag OCS operator expects to consume? `v${BRANCH_NAME}$git_hash`

Reply:
Yes correct, cc @Nikhil-Ladha ??

Reply:
There should be a hyphen in between the branch name and git hash, i.e, `v${BRANCH_NAME}-$git_hash`